src/Monofony/Plugin/FixturesPlugin/Recipe/config/sylius/fixtures.yaml

```diff
@@ -34,3 +34,4 @@ sylius_fixtures:
                             client:
                                 random_id: 5rbhrb0iiukokcwk8gow0w4ocgww0oco8g8gsgokwc0wcssg4w
                                 secret: 2rlxzhijcx448ow4c0gksw4wo8oo4k8kkwwg0osskk8g0k8kw8
+                                allowed_grant_types: [password, access_token, refresh_token]
```

Comment:
Let's try without and see if grant types are added.
```suggestion
```

Reply:
No they won't, if it is not present, the array is systematically empty.

Reply:
@stevemoreau I've tried too and default values does not work with array. So, could you remove this unnecessary changes and just keep the grant type values into the fixtures.yaml file please?